Description: Avoid remote fonts to avoid privacy breach
 Simply avoid custom fonts for now...
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2018-09-14
---
This patch header follows DEP-3: https://dep.debian.net/deps/dep3/
--- a/doc/API.htm
+++ b/doc/API.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>The Ghostscript Interpreter Application Programming Interface (API)</title>
     <!-- Supercedes the API in DLL.htm -->
--- a/doc/C-style.htm
+++ b/doc/C-style.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Ghostscript C Coding Guidelines</title>
     <!-- Originally: c-style.txt -->
--- a/doc/Commprod.htm
+++ b/doc/Commprod.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Conditions on Distributing Ghostscript in a Commercial Context</title>
     <!-- Originally: commprod.txt -->
--- a/doc/DLL.htm
+++ b/doc/DLL.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>How to Use the Ghostscript Dynamic Link Library (DLL)</title>
     <!-- Originally: dll.txt -->
--- a/doc/Deprecated.htm
+++ b/doc/Deprecated.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Deprecated Ghostscript Features</title>
 <link href="style.css" rel="stylesheet" type="text/css">
--- a/doc/Develop.htm
+++ b/doc/Develop.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Information for Ghostscript Developers</title>
 <link href="style.css" rel="stylesheet" type="text/css">
--- a/doc/Devices.htm
+++ b/doc/Devices.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Details of Ghostscript Output Devices</title>
     <!-- Originally: devices.txt -->
--- a/doc/Drivers.htm
+++ b/doc/Drivers.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>The Interface between Ghostscript and Device Drivers</title>
     <!-- Originally: drivers.txt -->
--- a/doc/Fonts.htm
+++ b/doc/Fonts.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Fonts and Font Facilities Supplied with Ghostscript</title>
     <!-- Originally: fonts.txt -->
--- a/doc/History9.htm
+++ b/doc/History9.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>History of Ghostscript versions 9.n</title>
     <!--
--- a/doc/Install.htm
+++ b/doc/Install.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>How to Install Ghostscript</title>
     <!-- Originally: install.txt -->
--- a/doc/Language.htm
+++ b/doc/Language.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Ghostscript and the PostScript Language</title>
     <!-- Originally: language.txt -->
--- a/doc/Lib.htm
+++ b/doc/Lib.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>The Ghostscript Library</title>
     <!-- Originally: lib.txt -->
--- a/doc/Make.htm
+++ b/doc/Make.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>How to Build Ghostscript from Source Code</title>
     <!-- Originally: make.txt -->
--- a/doc/News.htm
+++ b/doc/News.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Recent Changes in Ghostscript</title>
     <!-- Originally: NEWS -->
--- a/doc/Ps-style.htm
+++ b/doc/Ps-style.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Ghostscript PostScript Coding Guidelines</title>
 <link href="style.css" rel="stylesheet" type="text/css">
--- a/doc/Ps2epsi.htm
+++ b/doc/Ps2epsi.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Convert PostScript to Encapsulated PostScript Interchange Format</title>
     <!-- Originally: ps2epsi.txt -->
--- a/doc/Psfiles.htm
+++ b/doc/Psfiles.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>PostScript Files Distributed with Ghostscript</title>
     <!-- Originally: psfiles.txt -->
--- a/doc/Readme.htm
+++ b/doc/Readme.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Overview of Ghostscript</title>
     <!-- Originally: README -->
--- a/doc/Release.htm
+++ b/doc/Release.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>How to Prepare a Ghostscript Release</title>
     <!-- Originally: make.txt -->
--- a/doc/SavedPages.htm
+++ b/doc/SavedPages.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Ghostscript: Sample CMYK 32-bit Device that Supports Post Rendering Processing</title>
 <link href="style.css" rel="stylesheet" type="text/css">
--- a/doc/Source.htm
+++ b/doc/Source.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Guide to Ghostscript Source Code</title>
     <!-- Originally: source.txt -->
--- a/doc/Unix-lpr.htm
+++ b/doc/Unix-lpr.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Setting Up a Unix 1pr Filter for Ghostscript</title>
     <!-- Originally: unix-lpr.txt -->
--- a/doc/Use.htm
+++ b/doc/Use.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>How to Use Ghostscript</title>
     <!-- Originally: use.txt -->
--- a/doc/VectorDevices.htm
+++ b/doc/VectorDevices.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>High Level Output Devices</title>
 <link href="style.css" rel="stylesheet" type="text/css">
--- a/doc/WhatIsGS.htm
+++ b/doc/WhatIsGS.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Ghostscript: What Is Ghostscript?</title>
 <link href="style.css" rel="stylesheet" type="text/css">
--- a/doc/sample_downscale_device.htm
+++ b/doc/sample_downscale_device.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Ghostscript: Sample CMYK 32-bit Device that Supports Post Rendering Processing</title>
 <link href="style.css" rel="stylesheet" type="text/css">
--- a/doc/subclass.htm
+++ b/doc/subclass.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Ghostscript: Device Subclassing</title>
 <link href="style.css" rel="stylesheet" type="text/css">
--- a/doc/thirdparty.htm
+++ b/doc/thirdparty.htm
@@ -3,7 +3,7 @@
 <head>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+<!-- link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet" -->
 <link rel="shortcut icon" type="image/png" href="../../images/favicon.png">
 <title>Third Party Libraries Used by Ghostscript and GhostPDL</title>
     <!-- Originally: NEWS -->
